### ISSUE TITLE
Resize Song Editor Tracks by dragging them down

### ISSUE THREAD

**Issue by SpencerNagg** [LMMS/lmms#7198]
### Enhancement Summary

It took me a while to realize that tracks could be vertically resized by holding shift and dragging down. Shouldn't you also be able to just drag the line at the bottom of the track down to resize? Every other DAW I've tried does it like that, and it is more obvious - you might not know shift+drag does anything unless you read about it somewhere.

### Implementation Details / Mockup

I think the only thing that has to change visually is for the scroll mouse icon to appear on the line below the track.

![image](https://github.com/LMMS/lmms/assets/140673879/65e48be9-71a3-4e87-b422-458184605347)

### Please search the issue tracker for existing feature requests before submitting your own.

- [X] I have searched all existing issues and confirmed that this is not a duplicate.

**Comment by qnebra** [comment]
Possible by pointing cursor in line between tracks, pressing Shift and dragging mouse.

**Comment by SpencerNagg** [comment]
> Possible by pointing cursor in line between tracks, pressing Shift and dragging mouse.

I mentioned I'm aware of that in my comment. My point is that, aside from randomly trying modifier keys, no user is going to discover this feature inside the program because there's no visual indicator the tracks are resizable. Usually in other programs, either the edge of the track highlights as you drag over it or the cursor changes to indicate a resizable element.

**Comment by regulus79** [comment]
I agree, it would be better if some kind of cursor change happened, or if we also added a way to resize from the edge. Maybe I could look into doing that sometime.

**Comment by qnebra** [comment]
Ok
